src/app/content/components/ContentLink.tsx

```diff
@@ -64,6 +65,7 @@ export const ContentLink = (props: React.PropsWithChildren<Props>) => {
     ? createNavigationOptions({...search, ...systemQueryParams}, scrollTarget)
     : undefined;
   const URL = options ? relativeUrl + navigationOptionsToString(options) : relativeUrl;
+  const intl = useIntl();
```

Comment:
call `useServices` and pass the whole thing into showConfirmation, because pretty sure in future changes more services are needed